### DISCUSSION TITLE
Base64 encode returned token

### DISCUSSION THREAD

**Discussion by eydun** [Orange-OpenSource/hurl#1921] [Q&A] [answered]
My login-request returns a token, which I am able to capture with jsonpath.

But before I can use this token for authorization in the second request, I need to base64 encode it. Is that possible?

**Comment by jcamiel** [comment, marked as answer]
Hi @eydun, we have [filters](https://hurl.dev/docs/filters.html) that can process captures but unfortunately there is no base64 encode / decode filters. I'll fill an issue to add this filters.

As a workaround, depending on your use case, maybe you can use a [`[BasicAuth]` section](https://hurl.dev/docs/request.html#basic-authentication) that transforms some data to base64 (and creates an [`Authorization`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Authentication)

**Reply by eydun** [reply]
Thanks!

A base64-encode/decode-filter would be awesome.
